react-native-cli/index.js

```diff
@@ -42,10 +43,11 @@ if (cli) {
   switch (args[0]) {
   case 'init':
     if (args[1]) {
-      init(args[1]);
+      var verbose = process.argv.indexOf('--verbose') >= 0;
+      init(args[1], verbose);
     } else {
       console.error(
-        'Usage: react-native init <ProjectName>'
+        'Usage: react-native init <ProjectName> [--verbose]'
       );
       process.exit(1);
     }
```

Comment:
We need to avoid manually parsing the command line but use a library, similarly to what we're doing on the `private-cli`. Otherwise, things could break very easily and it will be harder and harder to implement as the number of options  this cl has increase. No need to change this pr, it's just a comment for the future. We may merge this cli with private-cli though. cc @frantic